spec/mailers/previews/alert_mailer_preview.rb

```diff
@@ -49,4 +57,9 @@ def example_alert(type: 'HighQualityArticleEditAlert')
     Alert.new(type: type, article: example_article,
               course: example_course, id: 9)
   end
+
+  def example_de_userfying_alert
+    Alert.new(type: 'DeUserfyingAlert', article: example_article,
+              course: example_course, id: 9, user: example_student)
```

Comment:
This didn't work for me, because the `example_course` doesn't get created but the alert object includes a user... so I get an error when trying to construct the contribution URL for the user. I fixed it locally by using `Course.last` in place of `example_course` here.

Reply:
I cannot replicate that, is there something for me to do ?

Reply:
Are you able to visit `/rails/mailers/alert_mailer/de_userfying_alert` without hitting an error?

A quick fix will just be to replace `example_course` with `Course.last` here, but I'm not sure about why it would work for you and not for me.

Reply:
Yes, there is no error showing up when accessing /rails/mailers/alert_mailer/de_userfying_alert
A bit strange.